ROMFS/px4fmu_common/init.d/airframes/4040_reaper

```diff
@@ -35,7 +35,6 @@ then
 	param set PWM_AUX_RATE 50
 	param set PWM_MIN 1100
 	param set PWM_MAX 1900
-	param set PWM_RATE 50
```

Comment:
@bkueng The reaper is a collective pitch quadcopter with servos.

Reply:
Dropped